.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -0,0 +1,33 @@
+---
+name: New Release
+about: Propose a Mew Release
```

Comment:
```suggestion
about: Propose a New Release
```

Reply:
:see_no_evil: I think it just can't be so small or short enough that I wouldn't be able to make a typo or other mistake. Thanks for spotting it !